ERCS/erc-7579.md

```diff
@@ -91,6 +91,12 @@ The account MAY also implement the following function in accordance with ERC-433
 function executeUserOp(PackedUserOperation calldata userOp, bytes32 userOpHash) external;
 ```
 
+If account chooses to implement `executeUserOp`, this method SHOULD execute `userOp.calldata` except 4 most significant bytes, which are reserved for `executeUserOp.selector` as per ERC-4337.
```

Comment:
is this too vague? should we specify that it MUST be delegatecalled like accounts do today?

Reply:
Yes, I was thinking of this as well. 
I agree, that delegatecall'ing here is preferred, but I don't see why it is a MUST.

IMO, what is important for the standard, is that exactly calldata[0:4] is executed, so we're always sure which part of calldata we can parse for validation parameters. 
So in theory it doesn't matter how account executes , it is more important what it executes

Reply:
in this case, it could eg be the case that calldata[4:24] is an address and calldata[24:] is calldata which is called; or calldata[4:36] is target and calldata[36:] is data; or any other way of encoding the data here; so this wouldnt really help much for validators right

Reply:
ok, I see your point. let me introduce changes